features/0309-didauthz/README.md

```diff
@@ -0,0 +1,309 @@
+# Aries RFC 0309: DIDAuthZ
```

Comment:
Why this title?  It's using DIDComm, but the authorization is coming from a verifiable credential. It seems like an odd name for this.  Again, not a big deal - I just don't get it.

Reply:
Noted.